source/class/qx/application/AbstractGui.js

```diff
@@ -101,9 +102,7 @@ qx.Class.define("qx.application.AbstractGui", {
     // interface method
     close(val) {
       // empty
-    },
-
-    // interface method
+    }, // interface method
```

Comment:
The `// interface method` applies to `terminate`, not to `close`. I think this whitespace was deleted accidentally.